src/shared/RangeInput.tsx

```diff
@@ -1,26 +1,49 @@
-import { Component } from "solid-js";
+import { Component, Show } from "solid-js";
 
-const RangeInput: Component = () => (
-  <div class="relative pt-1">
-    <label for="customRange2" class="form-label">
-      Example range
-    </label>
-    <input
-      type="range"
-      class="
-      form-range
-      h-6
-      w-full
-      appearance-none
-      bg-transparent
-      p-0
-      focus:shadow-none focus:outline-none focus:ring-0
-    "
-      min="0"
-      max="5"
-      id="customRange2"
-    />
-  </div>
-);
+const RangeInput: Component<{
```

Comment:
Are you up to trying to restyle the `RangeInput` component? At the moment, it renders differently on every browser and none of them conform to the design I plan to enforce down the line. Ideally I'm looking for something like this (pardon the low-res screenshot):

<img width="962" alt="Screen Shot 2023-02-08 at 16 52 09" src="https://user-images.githubusercontent.com/121980126/217636840-20e929e1-6934-4ac4-84f7-975fcc92df29.png">

But colored with the purple shade that everything else is using.

If not, just let me know since this is fine to merge as-is, it can be handled later down the line.

Reply:
updated the RangeInput component to be consistant across browsers and match the style of the screenshot provided (with the purple color)
![Screenshot_2023-02-08_18-55-00](https://user-images.githubusercontent.com/26354814/217678926-685d776f-f6b8-4b73-9bee-493a8d742e09.png)
![IMG_3181](https://user-images.githubusercontent.com/26354814/217678929-18e7b288-7859-45a2-beed-4c485dd48451.jpg)

Reply:
Awesome, thank you! I'll merge this in when the remaining conflicts are fixed.

Reply:
resolved conflicts